tests/e2e/sample-applications/mysql-persistent/storageclass/ocs-storagecluster-cephfs.yaml

```diff
@@ -0,0 +1,20 @@
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    description: Provides RWO and RWX Filesystem volumes
+    storageclass.kubernetes.io/is-default-class: "false"
+  name: ocs-storagecluster-cephfs
```

Comment:
This might conflict with what's already in cluster.

Are these just being added as reference?

It sounds like this storageclass should already be in the cluster where it would be available and should not be installed manually.

I would remove them from this PR or add a comment at the top of this file that it's for reference and to not apply it to a live cluster.

Reply:
https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html-single/deploying_openshift_data_foundation_using_amazon_web_services/index#:~:text=Verify%20that%20the%20following%20storage,openshift%2Dstorage.noobaa.io

Reply:
we are not asking folks to `oc create -f $this_file` it's just for reference.  MUTHA FRANKER :)

Reply:
/this-is-not-fine